website/client/components/challenges/findChallenges.vue

```diff
@@ -103,6 +103,16 @@ export default {
       let search = this.search;
       let filters = this.filters;
       let user = this.$store.state.user.data;
+
```

Comment:
This entire `filteredChallenges` function seems to be duplicate here and in `website/client/components/challenges/myChallenges.vue`, would you mind moving it to client/store/getters/challenges.js and then use just add it to this file with `mapGetters(...)`  like here https://github.com/HabitRPG/habitica/blob/cd92f44365d6dff4fe265b850a671e379369fe29/website/client/components/tasks/taskModal.vue#L599